tests/unit/routes/events/view/tickets-test.js

```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:events/view/tickets', 'Unit | Route | events/view/tickets', {
```

Comment:
Follow like how the other route tests are defined. (Use the custom `moduleFor` method)